### PR TITLE
New version

### PR DESCRIPTION
**Pull request by russellhoff**
Updated to the latest version

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -11,7 +11,7 @@
     <packaging>jar</packaging>
     <groupId>org.webjars</groupId>
     <artifactId>rgraph</artifactId>
-    <version>3_2015-07-27-stable-SNAPSHOT</version>
+    <version>RGraph4.66-stable</version>
```
Comment (jamesward) [inline pom.xml]: This should be `4.66-stable-SNAPSHOT`

Comment (russellhoff) [reply pom.xml]: Do you want me to made the change in the version number?

File: pom.xml [hunk 2]
```diff
@@ -11,7 +11,7 @@
     <packaging>jar</packaging>
     <groupId>org.webjars</groupId>
     <artifactId>rgraph</artifactId>
-    <version>3_2015-07-27-stable-SNAPSHOT</version>
+    <version>RGraph4.66-stable-SNAPSHOT</version>
```
Comment (jamesward) [inline pom.xml]: Remove the `RGraph` from this version.

**Comment by russellhoff** [comment]
done

**Pull request merged**